src/bower_components/emby-webcomponents/imageuploader/imageuploader.template.html

```diff
@@ -22,12 +22,13 @@ <h2 style="margin:0;">${HeaderAddUpdateImage}</h2>
                 <div class="imageEditor-dropZone fieldDescription">
                     <div>${LabelDropImageHere}</div>
                     <output id="imageOutput" class="flex align-items-center justify-content-center" style="position: absolute;top:0;left:0;right:0;bottom:0;width:100%;"></output>
-                    <input type="file" accept="image/*" id="uploadImage" name="uploadImage" style="position: absolute;top:0;left:0;right:0;bottom:0;width:100%;opacity:0;" />
+                    <input type="file" accept="image/x-png,image/jpeg" id="uploadImage" name="uploadImage" style="position: absolute;top:0;left:0;right:0;bottom:0;width:100%;opacity:0;" />
```

Comment:
Should be `image/png` since that's the official IANA media type.
https://www.iana.org/assignments/media-types/media-types.xhtml

Reply:
I'll include it, but also leave the x-, as it seems some software may still use that MIME. So I'll just be safe.